parser_library/src/processing/statement_providers/members_statement_provider.cpp

```diff
@@ -123,7 +123,7 @@ void members_statement_provider::fill_cache(
         reparsed_stmt.stmt = std::make_shared<semantics::statement_si_defer_done>(
             def_impl, std::move(op), std::move(rem), std::move(lits));
     }
-    cache.insert(processing_status_cache_key(status), std::move(reparsed_stmt));
+    cache.insert(processing_status_cache_key(status, ctx.hlasm_ctx->get_instruction_sets()), std::move(reparsed_stmt));
```

Comment:
probably need to have a test case for reparsing with changed configuration.

Reply:
Good point